src/Apps/Order2/Routes/Checkout/Components/Order2CollapsibleOrderSummary.tsx

```diff
@@ -0,0 +1,222 @@
+import ChevronDownIcon from "@artsy/icons/ChevronDownIcon"
+import { Box, Clickable, Flex, Image, Spacer, Text } from "@artsy/palette"
+import { RouterLink } from "System/Components/RouterLink"
+import createLogger from "Utils/logger"
+import type { Order2CollapsibleOrderSummary_order$key } from "__generated__/Order2CollapsibleOrderSummary_order.graphql"
+import type * as React from "react"
+import { useEffect, useState } from "react"
+import { graphql, useFragment } from "react-relay"
+
+const logger = createLogger("Order2CollapsibleOrderSummary")
+
+interface Order2CollapsibleOrderSummaryProps {
+  order: Order2CollapsibleOrderSummary_order$key
+}
+
+export const Order2CollapsibleOrderSummary: React.FC<
+  Order2CollapsibleOrderSummaryProps
+> = ({ order }) => {
+  const orderData = useFragment(FRAGMENT, order)
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  const artwork = orderData.lineItems[0]?.artwork
+  const artworkVersion = orderData.lineItems[0]?.artworkVersion
+
+  const requiredArtworkDataPresent =
+    artwork?.slug &&
+    artworkVersion?.image?.resized?.url &&
+    artworkVersion?.title &&
+    artworkVersion?.artistNames &&
+    artworkVersion?.date
+
+  const requiredOrderDataPresent =
+    orderData.buyerTotal?.display || orderData.itemsTotal?.display
+
+  // TODO: Handle missing/required data more intentionally.
+  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
+  useEffect(() => {
+    if (!requiredArtworkDataPresent) {
+      logger.error("Missing artwork data in Order2CollapsibleOrderSummary", {
+        artwork,
+        artworkVersion,
+      })
+    }
+    if (!requiredOrderDataPresent) {
+      logger.error("Missing order data in Order2CollapsibleOrderSummary", {
+        orderData,
+      })
+    }
+  }, [])
+
+  if (!requiredArtworkDataPresent) {
+    return null
+  }
+
+  const handleToggle = () => {
+    setIsExpanded(!isExpanded)
+  }
+
+  return (
+    <Box data-testid="OrderSummary" backgroundColor="mono0">
+      <Flex py={1} px={2} justifyContent="space-between">
+        <RouterLink flex={0} to={`/artwork/${artwork.slug}`} target="_blank">
+          <Image
+            mr={1}
```

Comment:
One tip with palette (this is semi-new, but has been the biggest boost in terms of overall UI-code organization) is to really leverage two things:

```
// horizontal things 
 <Flex gap={1}>
   thing 
   nextThing
 <Flex>
 ```
 
 And then vertical things can use 
 ```tsx
 <Stack gap={1}>
   ...
 </Stack>
 ```
 
 This allows us to get rid of Join (unless needing a divider or something), and also fixes the problems of join. 
 
 The main thing tho is i'm finding it has been incredibly effective in cleaning up mt, mb, mr, pr, etc props that are always littered all around components. Now we just define those gaps from the parent itself, and if any new children are added they automatically take advantage, and ui remains consistent

Reply:
(Not saying thats appropriate here -- just a heads up that these options are available)

Reply:
Oh, good to know. I just learned about `Stack`. This would only be for cases where there is a list of evenly-spaced elements?

Reply:
Yep. Similarly for Flex gap={1}, going rightwards. Most stuff is evenly placed, so good to lean into it